pipelines/short-read-alignment-vector/ShortReadAlignment.wdl

```diff
@@ -0,0 +1,467 @@
+version 1.0
+
+## Copyright Wellcome Sanger Institute, Oxford University and the Broad Institute 2020
+##
+## This WDL pipeline implements the Short Read Alignment Pipeline as described in
+## https://github.com/malariagen/pipelines/blob/master/docs/specs/short-read-alignment-vector.md
+##
+## This is an initial proof of concept implementation.  It is designed to ONLY work on one sample, with ONLY its
+## list of input fastqs.  It is currently implemented to run using Cromwell with a google cloud platform backend.
+##
+
+workflow ShortReadAlignment {
+  String pipeline_version = "0.0.1"
+
+  input {
+    Array[String] read_group_ids
+    Array[File] input_fastq1s
```

Comment:
These arrays were generated by taking one sample's set of inputs and pushing to the cloud.  Just structured like this for ease of (initial) testing.